app/components/Settings/Scheduler/loginModal.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Row, Column } from '../../layout.jsx';
+import Modal from '../../modal.jsx';
+
+const styles = {
```

Comment:
Not a blocking change, but these styles should go into its own file, e.g. `login-modal.css`; (having so many `<xxxx style={styles.xxxx}>` is a maintenance burden).